src/main.ts

```diff
@@ -230,6 +236,19 @@ export class Purchases {
     validateCurrency(params?.currency);
     const appUserId = this._appUserId;
     const offeringsResponse = await this.backend.getOfferings(appUserId);
+
+    const offeringIdFilter =
+      params?.offeringIdentifier === OfferingKeyword.Current
+        ? offeringsResponse.current_offering_id
+        : params?.offeringIdentifier;
+
+    if (offeringIdFilter) {
+      offeringsResponse.offerings = offeringsResponse.offerings.filter(
+        (offering: OfferingResponse) =>
+          offering.identifier === offeringIdFilter,
+      );
+    }
```

Comment:
One consequence of this is that if the given offering id is not the current, the response of the getOfferings will have a null `current` offering. This is probably fine, but maybe we should add it to the docs for the `current` field

Reply:
Done